### PR TITLE
Create your own distribution

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Create your own distribution
| Fixed ticket? | N/A

### CODE REVIEW

File: src/content/1.7/basics/installation/distribution.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Create your own distribution
```
Comment (PierreRambaud) [inline src/content/1.7/basics/installation/distribution.md:2]:
```suggestion
title: Create release
```

Shouldn't be something more generic? ping @matks

File: src/content/1.7/basics/installation/distribution.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+From the root path of your repository, execute:
+
+```shell
+php tools/build/CreateRelease.php --version="1.7.8.0"
+```
+
+## Modules
+
+If you want to embed some specific modules in your distribution, add it directly in the folder `modules/`. They will be automatically installed.
```
Comment (PierreRambaud) [inline src/content/1.7/basics/installation/distribution.md:17]:
```suggestion
If you want to embed some specific modules in your distribution, add them directly in the folder `modules/`. They will be automatically installed.
```

File: src/content/1.7/basics/installation/distribution.md (hunk 1 above)
Comment (Progi1984) [reply src/content/1.7/basics/installation/distribution.md:2]: I disagree. Release is not what I talk. Distribution is the subject.

Comment (Progi1984) [reply src/content/1.7/basics/installation/distribution.md:2]: I link to the build page for not duplicating some informations about creating a release.

**Review by matks — COMMENTED** [review]
I'm sorry I think this PR needs to be completed a little more 😅  it's too small and raises more questions than it answers

1. I don't think it should be in "Basics > Installation" section, building your own distribution is not a "basic" topic it's something advanced that will be performed by only a few people
2. It states that "to create a distribution you must create a release". Why? What is the link between these two? For a visitor, it's hard to understand the relationship between distribution and release.
3. "If you want to embed some specific modules in your distribution, add them directly in the folder `modules/`. They will be automatically installed." this needs to be explained a little further, how is the module installed: in the BO ? "installed" is even a bit ambiguous. When? by the installer ? which step?
4. What is a PrestaShop distribution? What would someone build one?

It looks like this page actually aims to provide some infos to someone who already has all these elements, so it's restricted for a few experienced users.

**Comment by kpodemski** [comment]
I'm missing a paragraph with something like:

> Why you might want to create your own distribution of the PrestaShop?

File: src/content/1.7/basics/installation/distribution.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+title: Create your own distribution
+menuTitle: Create your own distribution
```
Comment (eternoendless) [inline src/content/1.7/basics/installation/distribution.md:3]: You don't need menuTitle if it's the same as title
```suggestion
```

File: src/content/1.7/basics/installation/distribution.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+title: Create your own distribution
```
Comment (eternoendless) [inline src/content/1.7/basics/installation/distribution.md:2]:
```suggestion
title: Creating your own distribution
```

File: src/content/1.7/basics/installation/distribution.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+menuTitle: Create your own distribution
+weight: 35
+---
+
```
Comment (eternoendless) [inline src/content/1.7/basics/installation/distribution.md:6]:
```suggestion

# Creating you own PrestaShop distribution

```

File: src/content/1.7/basics/installation/distribution.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+For creating your own distribution, you must [create a release][create-release].
+
+## Modules
+
+If you want to embed some specific modules in your distribution, add them directly in the folder `modules/`. They will be automatically installed.
```
Comment (eternoendless) [inline src/content/1.7/basics/installation/distribution.md:11]: This one is recent, right?

File: src/content/1.7/basics/installation/distribution.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+---
+title: Create your own distribution
+menuTitle: Create your own distribution
+weight: 35
+---
+
+For creating your own distribution, you must [create a release][create-release].
```
Comment (eternoendless) [inline src/content/1.7/basics/installation/distribution.md:7]: I feel this needs a little more details. Why would someone create a distribution? Shouldn't we put this inside a new section (after installation) called "Deployment"? Or does it make sense that this is under "Basics" whereas this is an advanced feature?

File: src/content/1.7/basics/installation/distribution.md (hunk 6 above)
Comment (Progi1984) [reply src/content/1.7/basics/installation/distribution.md:11]: Affirmative. Thanks to @PierreRambaud

File: src/content/1.7/basics/installation/distribution.md (hunk 7 above)
Comment (Progi1984) [reply src/content/1.7/basics/installation/distribution.md]: Fixed

**Comment by matks** [comment]
Hello,

I have modified your PR to target the `1.7.x` branch.

We have modified the devdocs inner working in order to welcome multiple documentation versions. This is because soon we will have a 1.7 documentation and a 8.0 documentation, matching PrestaShop 1.7 or PrestaShop 8.0 softwares.

Read more on https://devdocs.prestashop.com/1.7/contribute/documentation/how/documentation-versions/

Please also note that now that we have multiple versions inside one documentation, we had to build a workflow that can handle this setup. This repository https://github.com/PrestaShop/docs now only contains the content of the documentation, and https://github.com/PrestaShop/devdocs-site contains the sources and the deployment workflow.

Because of this, some files have moved and you might have to rebase your Pull Request to match the new structure of the repository.

**Comment by Progi1984** [comment]
@matks Thank you for your feedback. I have targeted `8.x` as changes like module installation are available only on `8.0`.

(And if you have time for review).

**Review by NeOMakinG — APPROVED** [review]
Looks fine to me, and it's good to know

**Review by PierreRambaud — APPROVED** [review]

**Comment by eternoendless** [comment]
This PR has been merged in the wrong directory

**Pull request merged**